plugins/openwhisk/src/main/java/com/navercorp/pinpoint/plugin/openwhisk/OpenwhiskPlugin.java

```diff
@@ -0,0 +1,225 @@
+        });
+
+        transformTemplate.transform("whisk.common.tracing.NoopTracer$", new TransformCallback() {
+            @Override
+            public byte[] doInTransform(Instrumentor instrumentor, ClassLoader classLoader, String className, Class<?> classBeingRedefined, ProtectionDomain protectionDomain, byte[] classfileBuffer) throws InstrumentException {
+                final InstrumentClass target = instrumentor.getInstrumentClass(classLoader, className, classfileBuffer);
+
+                // add `apply` method interceptor
+                final MethodFilter applyMethodFilter = MethodFilters.chain(
+                        MethodFilters.name("setTraceContext")
+                );
+                for (InstrumentMethod method : target.getDeclaredMethods(applyMethodFilter)) {
+                    try {
+                        method.addInterceptor("com.navercorp.pinpoint.plugin.openwhisk.interceptor.NoopTracerSetTraceContextInterceptor");
+                        break;
+                    } catch (Exception e) {
+                        if (logger.isWarnEnabled()) {
+                            logger.warn("Unsupported method " + method, e);
+                        }
+                    }
+                }
+                return target.toBytecode();
+            }
+        });
+
+        transformTemplate.transform("whisk.common.TransactionId$", new TransformCallback() {
+            @Override
+            public byte[] doInTransform(Instrumentor instrumentor, ClassLoader classLoader, String className, Class<?> classBeingRedefined, ProtectionDomain protectionDomain, byte[] classfileBuffer) throws InstrumentException {
+                final InstrumentClass target = instrumentor.getInstrumentClass(classLoader, className, classfileBuffer);
+
+                // add `started` method interceptor
+                final MethodFilter startedMethodFilter = MethodFilters.chain(
+                        MethodFilters.name("started$extension")
+                );
+                for (InstrumentMethod method : target.getDeclaredMethods(startedMethodFilter)) {
+                    try {
+                        method.addInterceptor("com.navercorp.pinpoint.plugin.openwhisk.interceptor.TransactionIdStartedInterceptor");
+                    } catch (Exception e) {
+                        if (logger.isWarnEnabled()) {
+                            logger.warn("Unsupported method " + method, e);
+                        }
+                    }
+                }
+
+                // add `finished` method interceptor
+                final MethodFilter finishedMethodFilter = MethodFilters.chain(
+                        MethodFilters.name("finished$extension")
+                );
+                for (InstrumentMethod method : target.getDeclaredMethods(finishedMethodFilter)) {
+                    try {
+                        method.addInterceptor("com.navercorp.pinpoint.plugin.openwhisk.interceptor.TransactionIdFinishedInterceptor");
```

Comment:
If you are checking the number of method parameters after adding an Interceptor, you do not need to check the arguments.
If not, check the number of arguments. (to prepare for overloading .)